vite.config.js

```diff
@@ -27,5 +29,10 @@ export default defineConfig(({ command, mode }) => {
       port: HTTP_PORT,
       origin: env.PRIMARY_SITE_URL + ':' + originPort,
     },
+    plugins: [
+      viteStaticCopy({
+        targets: [{ src: 'src/icons/**/*', dest: './assets/icons' }],
+      }),
+    ],
```

Comment:
This is how we handle icons on MW and it works pretty well.

Open to other suggestions too. I considered putting them in the `public` directory.  But you [loose the ability to bundle SVG with your JS.](https://vitejs.dev/guide/assets#the-public-directory)